### ISSUE TITLE
Suggestion: Tab for import & export of UCP config

### ISSUE THREAD

**Issue by realPhint** [Sh0wdown/UnofficialCrusaderPatch#592]
Since you have to make sure every player has the exact same configuration of UCP running for multiplayer matches, maybe an additional tab in the GUI with "export settings" and "import settings" could be used to generate a file including your current settings that could then be sent to the other players who can then import that file with the "import settings" option to have the exact same settings chosen for them before installing. maybe this would make the sharing more intuitive.

**Comment by Krarilotus** [comment]
i like this idea!

**Comment by Truetobi91** [comment]
Maybe I'm missing something, but we do have a file that is being created that you can send someone so their patcher will have the exact same settings as you?

**Comment by GRhin** [comment]
Yes, but it's more confusing to explain to someone how to import.

**Comment by realPhint** [comment]
Thank you. Maybe I should have made it more clear that it's about the intuitive part. I think having the UCP functioning in Multiplayer is a very importent criteria for people to use it. Therefore designing this as easy to do as possible would be adviseable. Having an an export function to create and an import function to load into by chosing the file from a directory (like chosing the crusader folder when installing the UCP) is just something people are used to.
